src/main/urls.py

```diff
@@ -12,3 +12,7 @@
     path("", admin.site.urls),
     path("api-auth/", include("rest_framework.urls")),
 ]
+
+admin.site.site_title = "Affils Service"
```

Comment:
Is it normal to do this configuration in the `urls.py` module? Is it possible to do it in the `admin.py` module?

Reply:
But if it's normal/common to put it in `urls.py`, that's fine.

Reply:
I found in the docs that it can either be in ```urls.py``` or ```admin.py```. I have moved these lines into ```admin.py``` as requested.